src/nbl/video/COpenGLCommon.h

```diff
@@ -451,7 +451,7 @@ inline GLenum	getSizedOpenGLFormatFromOurFormat(IOpenGL_FunctionTable* gl, asset
 			break;
 	}
 #ifdef _NBL_DEBUG
-	os::Printer::log("Unsupported texture format", ELL_ERROR);
+	logger->log("Unsupported texture format", system::ILogger::ELL_ERROR);
```

Comment:
dont even bother loggin from those

Reply:
Oh btw, are these `#ifdef _NBL_DEBUG`s still needed?

Reply:
that whole case shouldn't be there, and I'm even wondering why somebody took the function table as an argument